spec.bs

```diff
@@ -4249,13 +4279,30 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
         is not null;
       * |ig|'s [=interest group/estimated size=] is greater than 50 KB.
     1. Set |ig|'s [=interest group/next update after=] to the [=current wall time=] plus 24 hours.
+    1. Set |ig|'s [=interest group/last updated=] to the [=current wall time=].
     1. [=list/Replace=] the [=interest group=] that has |ig|'s [=interest group/owner=] and
       [=interest group/name=] in the [=user agent=]'s [=interest group set=] with |ig|.
     1. <i id=abort-update>Abort update</i>: We jump here if some part of the
       [=interest group=] update failed. [=iteration/Continue=] to the next [=interest group=] update.
 
 </div>
 
+<div algorithm>
+
+To <dfn>Process updateIfOlderThanMs</dfn> given an [=origin=] |buyer|, and an [=ordered map=]
+|perInterestGroupData|:
+
+1. [=map/For each=] |igName| → |perIgData| of |perInterestGroupData|:
+  1. If |perIgData| doesn't [=map/contain=] "`updateIfOlderThanMs`", [=iteration/continue=].
+  1. Let |ig| be the [=interest group=] of the [=user agent=]'s [=interest group set=] whose
+    [=interest group/owner=] is |buyer| and whose name is |igName|:
+    1. If |ig| is not null and the [=current wall time=] minus |ig|'s [=interest group/last

```

Comment:
What if no IG exists with that owner and name, because it was deleted in between when interest groups were loaded for the auction and when we did the database update for updateIfOlderThanMs?

Reply:
Done.